README.md

```diff
@@ -51,6 +51,29 @@ Or install it yourself as:
 
 In the following sections, we explain the API provided by the gem and some basic usage.
 
+Many aspects of AR2DTO are [configurable for individual models](#setting-up-your-models); typically this is achieved by passing options to the has_dto method within a given model. Some aspects of AR2DTO are [configured globally](#global-configuration) for all models.
+
+### Global Configuration
+Global configuration options affect all threads and models where has_dto has been defined. A common place to put these settings is in a Rails initializer file such as `config/initializers/ar2dto.rb`.
+These settings are assigned directly on the `AR2DTO.configure` object.
+
+Configuration options are:
+ - default_excluded_attributes
+
+Syntax examples:
+
+```ruby
+  # config/initializers/ar2dto.rb
+
+  AR2DTO.configure do |config|
+    config.default_excluded_attributes = [:updated_at]
+  end
+
+  AR2DTO.configure.default_excluded_attributes = [:updated_at]
```

Comment:
I don't think we need to show this alternative. It can be confusing. Otherwise add an `OR` and separate the two examples